src/controllers/trip.controller.js

```diff
@@ -116,6 +116,24 @@ class TripController {
     ResponseService.setSuccess(200, 'List of all available locations', availableLocations);
     return ResponseService.send(res);
   }
+
+  /**
+ *
+ *
+ * @static
+ * @param {req} req
+ * @param {res} res
+ * @returns {response} @memberof Trips
+ */
+  static async editOpenTripRequest(req, res) {
+    const { userData } = req, userId = userData.id, { requestId, tripId } = req.params;
+    const [, [{ dataValues }]] = await TripService.updateTrip(
```

Comment:
is `tripId` not supposed to map to the `id` column in the database?

Reply:
No. Here `tripId` is the column in `trips table`

Reply:
1. On line 131, you mapped `requestId` to the `id` column in `trips` table.
2. On the same line, you mapped `tripId` to the `tripId` column in `trips` table.

By convention, the `id` column is the primary key in a table. That means a `user` table which has an `id` column will have `userId` as a foreign key which will be referenced in another table.

On line 131, there is a mismatch and that can cause confusion. `requestId` should not map to the `id` column of the `trips` table and there shouldn't even be a `tripId` in the `trips` table but `id`.

Reply:
@gadishimwe  created this additional tripId to be able to identify the trips created as multi-city. its their unique identifier

Reply:
Then the column name should be `parentId`. Using `tripId` is confusing

Reply:
What is `parentId`. we don't have that column in all database tables.

Reply:
@higustave-ops Please explain why a `tripId` is set on the trips table for multi-city trips

Reply:
@adesege can we have a short call to explain to you well?

Reply:
Sure